package.json

```diff
@@ -45,8 +45,9 @@
     "@babel/register": "^7.4.4",
     "@binary-com/binary-indicators": "^1.6.3",
     "app-module-path": "^2.2.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.8.0",
+    "babel-jest": "^23.4.2",
```

Comment:
(question) Why this change was needed?

Reply:
Oh I see it was advised, but I was wondering why `babel-jest` is downgraded.

Reply:
It is needed to pass Circle-ci tests. Aaron advised me to do so.

Reply:
Honestly I'm not sure. It was automatically changed after updating babel